src/MetaMorpho.sol

```diff
@@ -285,13 +285,15 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         }
     }
 
-    function disableMarket(Id id) external onlyCuratorRole {
+    /// @notice Submits a forced market removal from the vault, losing all funds supplied to the market.
+    /// @dev To remove it from the withdraw queue after the timelock, the market also needs to have a zero cap.
+    /// @dev Warning: Submitting a forced removal will overwrite the timestamp at which the market will be removable.
+    function submitMarketRemoval(Id id) external onlyCuratorRole {
         if (config[id].removableAt != 0) revert ErrorsLib.AlreadySet();
-        if (config[id].cap == 0) revert ErrorsLib.MarketNotEnabled();
```

Comment:
why don't we do the opposite ie we require that the cap is 0?

Reply:
discussion merged with: https://github.com/morpho-org/metamorpho/pull/312#discussion_r1386302217